src/components/image/image.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import invariant from "invariant";
+import { StyledImage, StyledImageProps } from "./image.style";
+
+export const Image = ({ alt, src, children, ...rest }: StyledImageProps) => {
+  invariant(
+    !src || (src && !children),
```

Comment:
```suggestion
    !src || !children,
```

Comes to the same thing but imo it's easier to understand this way.

Reply:
right, I got carried away when reversing that conditional for invariant